### ISSUE TITLE
Link to IoT Hub functionality

### ISSUE THREAD

**Issue by gcrev93** [great-lakes/project-egypt#13]
- Use Azure APIs to access IoT Hubs after user has entered IoT Hub connection string

**Comment by saroup** [comment]
if I understand correctly, this task would go under adding a hub. The success criteria for adding a hub is being able to successfully calling the iothub apis. (I don't think this should be a separate issue)

**Comment by gcrev93** [comment]
Adding a hub is in the backlog.. and that is a diff functionality . That is the function of actually creating an IoThub from the site.. not just copy and pasting in a connection string @saroup

**Comment by gcrev93** [comment]
Yes! thank you for clarifying. I will edit the other issue so help differentiate the two @saroup
